games-engines/openxcom/openxcom-1.0.0_p20180317-r1.ebuild

```diff
@@ -1,61 +1,69 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-inherit cmake-utils gnome2-utils
+EAPI=7
 
-COMMIT="ea9ac466221f8b4f8974d2db1c42dc4ad6126564"
-DESCRIPTION="An open-source reimplementation of the popular UFO: Enemy Unknown"
+inherit cmake xdg-utils
+
+DESCRIPTION="Open-source reimplementation of the popular UFO: Enemy Unknown"
 HOMEPAGE="https://openxcom.org/"
-SRC_URI="https://github.com/SupSuper/OpenXcom/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/SupSuper/OpenXcom.git"
+else
+	COMMIT="ea9ac466221f8b4f8974d2db1c42dc4ad6126564"
+	SRC_URI="https://github.com/SupSuper/OpenXcom/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64 ~x86"
+	S="${WORKDIR}/OpenXcom-${COMMIT}"
+fi
 
 LICENSE="GPL-3+ CC-BY-SA-4.0"
 SLOT="0"
-KEYWORDS="~amd64 ~arm64 ~x86"
 IUSE="doc"
 
-RDEPEND=">=dev-cpp/yaml-cpp-0.5.1
+RDEPEND="
+	>=dev-cpp/yaml-cpp-0.5.1
 	media-libs/libsdl[opengl,video]
 	media-libs/sdl-gfx
 	media-libs/sdl-image[png]
 	media-libs/sdl-mixer[flac,mikmod,vorbis]"
-DEPEND="${RDEPEND}
-	doc? ( app-doc/doxygen )"
+DEPEND="${RDEPEND}"
+BDEPEND="doc? ( app-doc/doxygen )"
 
-S="${WORKDIR}/OpenXcom-${COMMIT}"
 DOCS=( README.md )
 
 src_compile() {
-	use doc && cmake-utils_src_compile doxygen
-	cmake-utils_src_compile
+	use doc && cmake_build doxygen
+	cmake_src_compile
```

Comment:
does the doc stuff have to be before the main stuff? Also, I generally prefer putting the `HTML_DOCS` definition in the same block as the `use doc`, but again, your call